setup.sh

```diff
@@ -0,0 +1,382 @@
+          chmod +x $DOCKER_COMPOSE_BIN
+          echo
+          writeBold "$GREEN" "[✔] Docker Compose successfully installed."
+        else
+          writeBold "$RED" "[✖] curl or wget need to be installed to launch the Docker Compose installation script,"
+          writeBold "$RED" "    but none seems to be installed on your system."
+          echo
+          writeBold "$BLUE" "Please install curl or wget and re-run this script."
+          echo
+          exit 3
+        fi
+        ;;
+      [nN] | '')
+        echo
+        writeBold "$BLUE" "Ok. Please install Docker Compose and re-run this script."
+        echo
+        exit 0
+        ;;
+      *)
+        echo
+        writeBold "$RED" "[✖] I did not understand your answer."
+        ;;
+    esac
+  done
+}
+
+setupMapCount() {
+  SYSCTL_CONF_FILE=/etc/sysctl.conf
+  echo
+  writeBold "[ℹ] The kernel configuration variable vm.max_map_count must be set to at least $REQUIRED_MAP_COUNT"
+  writeBold "    for Kuzzle to work properly, but it seems to be set to $MAP_COUNT on your system."
+  writeBold "    This script can set it automatically or you can do it manually."
+  write     "    More information at https://www.elastic.co/guide/en/elasticsearch/reference/5.x/vm-max-map-count.html"
+  while [[ "$setVmParam" != [yYnN] ]]
+  do
+    promptBold "[❓] Do you want to set the vm.max_map_count now? (y/N) "
+    read setVmParam trash
+    case "$setVmParam" in
+      [yY])
+        echo
+        writeBold "Setting kernel variable vm.max_map_count to $REQUIRED_MAP_COUNT..."
+        sysctl -w vm.max_map_count=$REQUIRED_MAP_COUNT
+        if [ -z "$MAP_COUNT" ]; then
+          echo "vm.max_map_count=$REQUIRED_MAP_COUNT" > $SYSCTL_CONF_FILE
+        else
+          sed 's/vm.max_map_count=.+/vm.max_map_count=$REQUIRED_MAP_COUNT/g' $SYSCTL_CONF_FILE > $SYSCTL_CONF_FILE
+        fi
+        echo
+        writeBold "$GREEN" "[✔] Kernel variable successfully set."
+        ;;
+      [nN] | '')
+        echo
+        writeBold "$BLUE" "Ok. Please set the kernel variable and re-run this script."
+        echo
+        exit 0
+        ;;
+      *)
+      echo
+      writeBold "$RED" "[✖] I did not understand your answer."
+      ;;
+    esac
+  done
+}
+
+collectPersonalData() {
+  echo
+  writeBold  "[ℹ] Please let us know a little bit about youself."
+  promptBold "    What's your email address?${NORMAL} (press Enter to skip)"
+  read email trash
+  promptBold "    What's your name?${NORMAL} (press Enter to skip)"
+  read firstName lastName otherName yetAnotherName trash
+  promptBold "    What do you plan to use Kuzzle for?${NORMAL} (press Enter to skip)"
+  read purpose
+  # TODO send collected data to analytics service
+}
+
+startKuzzle() {
+  composerYMLURL="https://raw.githubusercontent.com/kuzzleio/kuzzle-build/master/docker-compose/kuzzle-docker-compose.yml"
```

Comment:
Since github already changed its URL format once, wouldn't it be safer to use our alias? (`http://kuzzle.io/docker-compose.yml`)

I let you decide on this one.